tests/core/calculators/mrcc/test_io.py

```diff
@@ -0,0 +1,333 @@
+from __future__ import annotations
+
+import io
+from io import StringIO
+from pathlib import Path
+
+import pytest
+from ase.atoms import Atoms
+from ase.calculators.calculator import compare_atoms
+
+from quacc.calculators.mrcc.io import (
+    read_energy,
+    read_geom_mrccinp,
+    read_mrcc_output,
+    read_mrcc_outputs,
+    write_mrcc,
+)
+
+
+def test_read_geom_mrccinp():
+    reference_inputfile = """calc=LNO-CCSD(T)
+basis=cc-pVDZ
+charge=0
+mult=1
+geom=xyz
+3
+
+H   1.0 0.0 0.0
+H   2.0 0.0 0.0
+O   3.0 0.0 0.0
+"""
+    atoms = read_geom_mrccinp(io.StringIO(reference_inputfile))
+    atoms_ref = Atoms(
+        "H2O", positions=[[1.0, 0.0, 0.0], [2.0, 0.0, 0.0], [3.0, 0.0, 0.0]]
+    )
+    assert not compare_atoms(atoms, atoms_ref, tol=1e-7)
+
+
+def test_write_mrcc(tmpdir):
+    atoms = Atoms("H2O", positions=[[1.0, 0.0, 0.0], [2.0, 0.0, 0.0], [3.0, 0.0, 0.0]])
+    atoms.set_tags([71, 71, 0])
+
+    params = {
+        "charge": 0,
+        "mult": 1,
+        "mrccinput": {
+            "calc": "LNO-CCSD(T)",
+            "basis": "cc-pVDZ",
+            "symm": "off",
+            "localcc": "on",
+            "lcorthr": "normal",
+            "ccprog": "ccsd",
+            "ccsdalg": "dfdirect",
+            "dfbasis_cor": "cc-pVDZ-RI",
+        },
+        "mrccblocks": """scfmaxit=1000
+usedisk=0""",
+    }
+    write_mrcc(Path(f"{tmpdir}/MINP"), atoms, params)
+
+    with open(f"{tmpdir}/MINP") as fd:
+        generated_inputfile = fd.readlines()
```

Comment:
Here and elsewhere, we should do `tmp_path / "MINP"` rather than string concatenation.